doc/releases/release-notes-4.0.rst

```diff
@@ -505,6 +680,9 @@ Drivers and Sensors
     * Added Würth Elektronik HIDS-2525020210002
       :dtcompatible:`we,wsen-hids-2525020210002` humidity sensor driver.
 
+  * Added general samples for triggers
+  * Added driver for NXP's fxls8974 accelerometer
+
 * Serial
 
```

Comment:
missing spi changes

Reply:
What are the spi changes that are missing?

Reply:
unless I'm missing something, I don't see any notes on spi changes in this PR

Reply:
there was at least the change that fixed the lpspi native (not gpio) chip selelct when using syncrhonous DMA api on MCXN, that might be worth mentioning

Reply:
not fixed yet